recipes/aiochclient/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "aiochclient" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4b686be9fa140e5c5fc3ff77062ca91e8b521d2df3efd31868f032f773befb13"
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+  host:
+    - cython
+    - pip
+    - python >=3.6
```

This version constraint should be removed as you have `skip`:
```suggestion
    - python
```
Constraining the version directly **_only_** works properly with `noarch: python` packages.